volume/store/store.go

```diff
@@ -309,16 +309,20 @@ func (s *VolumeStore) create(name, driverName string, opts, labels map[string]st
 // This makes sure there are no races between checking for the existence of a volume and adding a reference for it
 func (s *VolumeStore) GetWithRef(name, driverName, ref string) (volume.Volume, error) {
 	name = normaliseVolumeName(name)
```

Comment:
What is the intent of the changes in store.go? Are you trying to fix all callsites of Locker.Lock to set appropriate errors? If so, there's more (eg, libcontainer/client.go and even VolumeStore's Get(), CreateWithRef(), etc) and I'd recommend doing that in a separate PR.